pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -4742,6 +4739,43 @@ snapshots:
     icon: icon-search
     heading: No snapshots found
     body: Click on Create Snapshot to get started
+  dialog:
+    restore:
+      header: Are you sure you want to restore this Snapshot?
+      info: 'Please be aware that the current installation, including preferences, will be replaced. If the preference window is open it will be automatically closed and all unsaved changed discarded.'
+      actions:
+        ok: 'Restore'
+        cancel: 'Cancel'
+    delete:
+      header: Are you sure you want to delete this Snapshot?
+      info: 'text'
+      actions:
+        ok: 'Delete'
+        cancel: 'Cancel'
+    restoring:
+      header: 'Restoring {snapshot}'
+      message: "Please wait some minutes for the Snapshot restoration.<br/>The time depends on your machine resources.<br/>While creating the snapshot Rancher Desktop will be unavailable.<br/><br/>Once the process is finished the VM will restart, with the '<b>'{snapshot} active'</b>'."
```

Comment:
```suggestion
      message: "Please wait while snapshot {snapshot} is restored. This may take some time, depending on your machine's resources.<br/><br/>Rancher Desktop will be temporarily unavailable during this operation. Once completed, the VM will restart with the '<b>'{snapshot}'</b>' snapshot active."
```